geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/java/org/locationtech/geomesa/fs/storage/common/interop/ConfigurationUtils.java

```diff
@@ -35,6 +36,11 @@ public static void setLeafStorage(SimpleFeatureType sft, boolean leafStorage) {
         sft.getUserData().put("geomesa.fs.leaf-storage", String.valueOf(leafStorage));
     }
 
+    public static void setTargetFileSize(SimpleFeatureType sft, String size) {
+        Memory$.MODULE$.bytes(size).get(); // validate input
+        sft.getUserData().put("geomesa.fs.file-size", size);
```

Comment:
While I see this is consistent, is there any way to avoid the magic string and reference the static variable instead?

Reply:
i spent about 15 minutes trying to get this to work and couldn't... supposedly `org.locationtech.geomesa.fs.storage.common.package$StorageKeys$.MODULE$.FileSizeKey()` should work but that doesn't compile, even though the class shows up in scalap and javap. meh